src/guide/views/pagination/PaginationExample.jsx

```diff
@@ -0,0 +1,56 @@
+
+import React, {
+  Component,
+  PropTypes,
+} from 'react';
+
+import Page, {
+  Example,
+} from '../../components/page/Page.jsx';
+
+import {
+  Pagination,
+} from '../../../framework/framework';
+
+export default class PaginationExample extends Component {
+
+  constructor() {
+    super();
```

Comment:
Let's pass the props though. I know it is not needed on React 15 but it is on React 16, so if we add it now when we migrate one less thing to update.

```
constructor(props) {
    super(props);
```